net/dhtd/Makefile

```diff
@@ -0,0 +1,46 @@
+include $(TOPDIR)/rules.mk
+
+
+PKG_NAME:=dhtd
+PKG_VERSION:=0.1.0
+PKG_RELEASE:=1
+
+PKG_LICENSE:=MIT
+
+PKG_SOURCE_URL:=https://codeload.github.com/mwarning/dhtd/tar.gz/v$(PKG_VERSION)?
+PKG_SOURCE:=dhtd-$(PKG_VERSION).tar.gz
+PKG_HASH:=13c3a2c5ce5d35d5aa1b156b13dcc489456ee61efc3020dff5e34d18843666ef
+PKG_BUILD_DIR:=$(BUILD_DIR)/dhtd-$(PKG_VERSION)
```

Comment:
I think this  is not necessary. We already set this if `PKG_NAME` and `PKG_VERSION` is set.
See https://github.com/openwrt/openwrt/blob/master/include/package.mk#L11